src/react/deckgl.js

```diff
@@ -127,7 +127,13 @@ export default class DeckGL extends React.Component {
       ref: c => (this.overlay = c),
       key: 'overlay',
       id,
-      style: Object.assign({}, style, {position: 'absolute', left: 0, top: 0, width, height})
+      style: Object.assign({}, style, {
```

Comment:
I guess we could just put the `style` object last in the `Object.assign`,  to give the app full control, rather than "double assign" each param like this.

Reply:
will do, was not sure whether the override was done on purpose.